src/vanta_ledger/routes/notifications.py

```
@@ -0,0 +1,13 @@
+from fastapi import APIRouter, Depends
+from typing import List
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)